src/azure-cli/azure/cli/command_modules/appservice/_help.py

```diff
@@ -2471,6 +2471,46 @@
       text: az staticwebapp appsettings delete -n MyStaticAppName --setting-names key1 key2
 """
 
+helps['staticwebapp identity'] = """
+type: group
+short-summary: Manage a static web app's managed service identity
```

Comment:
It is recommended to use the name `managed identity` instead of `managed service identity`, because 
> Managed identities for Azure resources is the new name for the service formerly known as Managed Service Identity (MSI).

doc link: https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview